README.md

```diff
@@ -302,19 +302,28 @@ The above walkthrough follows the actual implementation process used to produce
 
 ## Manage the operator using the Operator Lifecycle Manager
 
-The previous section has covered manually running an operator. In the next sections, we will explore using the operator Lifecycle Manager which is what enables a more robust deployment model for operators being run in production environments.
+The previous section has covered manually running an operator. In the next sections, we will explore using the [Operator Lifecycle Manager][operator_lifecycle_manager] (OLM) which is what enables a more robust deployment model for operators being run in production environments.
+
+OLM helps you to install, update, and generally manage the lifecycle of all of the operators (and their associated services) on a Kubernetes cluster. It runs as an Kubernetes extension and lets you use `kubectl` for all the lifecycle management functions without any additional tools.
+
 
-The Operator Lifecycle Manager helps you to install, update, and generally manage the lifecycle of all of the operators (and their associated services) on a Kubernetes cluster. It runs as an Kubernetes extension and lets you use `kubectl` for all the lifecycle management functions without any additional tools.
 
 ### Generate an operator manifest
 
-The first step to leveraging the Operator Lifecycle Manager is to create a manifest. An operator manifest describes how to display, create and manage the application, in this case memcached, as a whole. It is required for the Operator Lifecycle Manager to function.
+The first step to leveraging OLM is to create a [Cluster Service Version][csv_design_doc] (CSV) manifest. An operator manifest describes how to display, create and manage the application, in this case memcached, as a whole. It is required for OLM to function.
+
+The Operator SDK CLI can generate CSV manifests via the following command:
+```console
+$ operator-sdk olm-catalog gen-csv --csv-version 0.0.1
+```
+For more details see the SDK's [CSV generation doc][csv_generation_doc].
+
 
-For the purpose of this guide, we will continue with this [predefined manifest][manifest_v1] file for the next steps. If you’d like, you can alter the image field within this manifest to reflect the image you built in previous steps, but it is unnecessary. In the future, the Operator SDK CLI will generate an operator manifest for you, a feature that is planned for the next release of the Operator SDK.
+For the purpose of this guide, we will continue with this [predefined manifest][manifest_v1] file for the next steps. If you’d like, you can alter the image field within this manifest to reflect the image you built in previous steps, but it is unnecessary.
 
 ### Deploy the Operator
 
-Deploying an operator is as simple as applying the operator’s manifest to the desired namespace in the cluster.
+Deploying an operator is as simple as applying the operator’s CSV manifest to the desired namespace in the cluster.
```

Comment:
How is this different from applying the deployment before? There should be some mention of cleanup from the previous steps that applied the rbac, crd, and deployment for the operator.

Reply:
This is a general comment, I'm thinking out loud. This can be for a future PR. :slightly_smiling_face:

Reply:
Not much different since we're directly applying the CSV and expecting to users to fulfill the RBAC and CRD requirements manually.
I think this is another reason to update the guide to use CatalogSources and Operator Registry.

I'll add a cleanup sub-section at the end of the SDK section however so the pre-reqs are not already installed.